docs/Using-Operations-Service.md

```diff
@@ -182,12 +182,19 @@ After that, you can produce an off-line signature using the following code:
 ```swift
 import WultraMobileTokenSDK
 
-func approveQROperation(operation: WMTQROperation, password: String) {
+func approveQROperation(operation: WMTQROperation, password: String?) {
 
     let authentication = PowerAuthAuthentication()
     authentication.usePossession = true
-    authentication.usePassword = password
-    authentication.useBiometry = false
+    if let password = password {
```

Comment:
dtto as with Android - the code is likely correct but may lead the developer on the wrong path...